src/main/java/com/spotinst/sdkjava/model/ISpotOceanEcsClusterRepo.java

```diff
@@ -0,0 +1,7 @@
+package com.spotinst.sdkjava.model;
```

Comment:
General comment:
1. in your tests try to create entity and then retrieve it in debug and compare between the object that send to the create method and the one that retrieved
2. check the update for a several fields, Please ask Shibel after he will comback from his vacation to give you handover for the partial update mechanism (for a massive creation for tests that update entity and verify that the entity updated

Reply:
1. Tested Object that is being send and retrieved after create method. 
2. Update - Currently Tested all possible attributes values with updated and verified it's updated.  So changed few attribute values in example file also. 
Regarding update Partial Mechanism - Yes, I will check with Shibel once he is back from vacation.